File: help/eng_gb/exercicios_da_ficha_9/Ficha09_008/Ficha09_008.py
# Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0


print( "Fibonaci sequence sum\n " , end="")
f0 = 1 
f1 = 1 
print(f0 ,  " ", f1  , end="")
sum = 2 
while f0 + f1 < 100  :
    fn = f0 + f1 
    sum = sum + fn
    print( " ", fn  , end="")
    f1 = f0
    f0 = fn
print( "\nSum = ", sum  , end="")


#Python 3  - Automatic translated by Algorithmi 22.05
#(c) Ant0nio M@nso 	Sat, 2022 Oct 01 00:11

File: help/eng_gb/exercicios_da_ficha_9/Ficha09_008/Ficha09_008.php
<?php
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    echo  "Fibonaci sequence sum\n " ;
    $f0 = 1 ;
    $f1 = 1 ;
    echo $f0  .  " " . $f1  ;
    $sum = 2 ;
    while($f0 + $f1 < 100 ){
        $fn = $f0 + $f1 ;
        $sum = $sum + $fn ;
        echo  " " . $fn  ;
        $f1 = $f0 ;
        $f0 = $fn ;
    }
    echo  "\nSum = " . $sum  ;
}

main();

//PHP 5.3 +  - Automatic translated by Algorithmi 22.05
//(c) Luís Lopes 	Sat, 2022 Oct 01 00:11

?>